projects/observability/src/public-api.ts

```diff
@@ -388,3 +388,11 @@ export * from './shared/components/bar-gauge/bar-gauge.module';
 
 // Time Range utils
 export * from './shared/utils/time-range';
+
+// Custom Dashboard
+export * from './pages/custom-dashboard/custom-dashboard.component';
+export * from './pages/custom-dashboard/custom-dashboard.module';
```

Comment:
This shouldn't be present in the PR.
In my opinion, this PR should only introduce `iframe` component.